### ISSUE TITLE
Steam Achievement Interaction

### ISSUE THREAD

**Issue by Spoonweaver** [ohrrpgce/ohrrpgce#1152]
A good amount of people have been putting their games on Steam and being able to interact with the achievement system on there would be great! I think it would also be great to just have the ability to interact with systems LIKE steam's achievements as I think that could open up a lot of possibilities.

**Comment by rversteegen** [comment]
Oh! Released in Ichorescent, thanks to Mike.

The achievements system is flexible enough to support achievements in the Blackbox (Xbox/PS/Switch) ports too.

**Comment by rversteegen** [comment]
OF course, this request was pretty inspecific, and we do still want to also add script commands for interacting with achievements. Feel free discuss that here or in a new issue.
